api/src/plugins/jwt.ts

```diff
@@ -20,13 +36,22 @@ const jwt: FastifyPluginCallback = async (fastify, opts, done) => {
 
   fastify.register(fastifyJwt, {
     secret: process.env.JWT_SECRET,
```

Comment:
Utiliser deux secrets différents pour access et refresh

Reply:
It will be corrected in a futur PR